server/server.js

```diff
@@ -61,5 +61,8 @@ app.use('/api/user', userRead);
 app.use('/api/groups', tokenService.verifyToken, groups);
 
 //process.env["NODE_TLS_REJECT_UNAUTHORIZED"] = 0;
+app.get('/', function (req, res) {
```

Comment:
was this added for some kind of testing?

Reply:
Yes. I am removing it